decisions/0015-manage-short-lived-access-tokens-in-tests.md

```diff
@@ -0,0 +1,36 @@
+# 15. Manage short-lived access tokens in tests
+
+Date: 2019-01-16
+
+## Status
+
+Accepted
+
+## Context
+
+We have implemented the VCR gem as part of our testing suite; this gem allows you to "record your test suite's HTTP interactions and replay them during future test runs for fast, deterministic, accurate tests."
```

Comment:
I think we need to be explicit here that we've been committing the cassettes into the repo - without that context some of the options below don't make sense.